src/Confluent.Kafka/project.json

```diff
@@ -14,8 +14,7 @@
     },
 
     "frameworks": {
-        "net451": { },
```

Comment:
these shouldn't change. we should continue to target the lowest version of netstandard we can and only increase this for good reason. and unfortunately we need to target net451 separately as this this is not compatible with netstandard1.3. changing to netcoreapp1.0 means we can't link with .net framework apps.

more info on netstandard versions.
https://blogs.msdn.microsoft.com/dotnet/2016/09/26/introducing-net-standard/

unfortunately I don't know any way to make dotnet happy with this configuration on linux/mac. I have a script  that allows me to comment/uncomment net451 (yes, in a JSON file, but dotnet doesn't complain) which i run before and after checking in (have a terminal window dedicated to running this script). There must be a better way... surely. 

```
#!/usr/local/bin/python

with open ("/git/confluent-kafka-dotnet/src/Confluent.Kafka/project.json", "r") as myfile:
        data = myfile.readlines()

out = []
for line in data:
        line2 = line[:-1]
        if "net451" in line2:
                if line2[0:2] == "/*":
                        line2 = line2[2:-2]
                else:
                        line2 = "/*" + line2 + "*/"
        out.append(line2)

with open ("/git/confluent-kafka-dotnet/src/Confluent.Kafka/project.json", "w") as myfile:
        for line in out:
                myfile.write(line + "\n")
```